src/components/node/stanford-page/page-display.tsx

```diff
@@ -13,11 +13,11 @@ const StanfordPage = async ({node, ...props}: {node: NodeStanfordPage}) => {
   })
 
   return (
-    <article {...props}>
+    <div {...props}>
```

Comment:
@pookmish we're running into this html validation error on some of the pages since this article wrapper isn't associated with an `aria-labelledby` and usually the nested H2 is `aria-labelledby` to a section wrapper.

https://validator.w3.org/nu/?doc=https%3A%2F%2Flibrary.stanford.edu%2Flibraries%2Fbranches-and-centers

Any thoughts? 👀

Reply:
Sounds good @pookmish ! For now I updated the other page-displays to `<div>`s.

 I created this [ticket](https://stanfordits.atlassian.net/browse/SUL23-840) to circle back and refactor this to keep things more in sync across our decoupled sites and for maintainability 👍 I'll tackle the refactor in the next sprint, after Events & Exhibits.